cmd/xl-v1-metadata.go

```diff
@@ -109,10 +109,11 @@ func BitrotAlgorithmFromString(s string) (a BitrotAlgorithm) {
 // objectPartInfo Info of each part kept in the multipart metadata
 // file after CompleteMultipartUpload() is called.
 type objectPartInfo struct {
-	Number int    `json:"number"`
-	Name   string `json:"name"`
-	ETag   string `json:"etag"`
-	Size   int64  `json:"size"`
+	Number     int    `json:"number"`
+	Name       string `json:"name"`
+	ETag       string `json:"etag"`
+	Size       int64  `json:"size"`
+	ActualSize int64  `json:"actualSize"`
```

Comment:
Just a naming thing - but can we use a "better" name than `actualSize` - maybe `decompressedSize` to directly indicate for what it's used? `actualSize` is very generic...
Same for `ObjectInfo.ActualSize`...

Reply:
This is on purpose to avoid compression specific names.

Reply:
So we're going to use `ActualSize` for any functionality that requires to remember the original size of the object?! If so okay, however notice that all these use cases cannot be combined in this case. For example (assume feature X requiring `ActualSize` field) we cannot use Compression and the feature `X` for the same object. 

So we stay generic but we can never use the field in the future if it conflicts...

Reply:
We may re-use the actual size.
Actual size is just a knowledge here in handler which we merely share it to the object layer so that it can use/reuse for compression or any size-variant features coming ahead.
So i think it wont be conflicting since the meaning for `actual-size` will be same for any use cases. 
Any thoughts on this @aead ?

Reply:
So your saying `actual-size` *always* represents the size of the uploaded object without any processing?! Than I guess it's okay.
If `actual-size` refers to something else - e.g.:
1. Upload: `size: 5 MB, actual-size: 5MB`
2. Compression: `size 3 MB, actual-size: 5MB`
3. Feature-X: `size 3.2 MB, actual-size: 5.2MB`

Than we may get conflict issues depending on the features using `actual-size`.
But if all features using `actual-size` satisfy: 
`f(object, size, actual-size) -> (object', size' actual-size)` it's okay.

Reply:
Yes @aead  ! `actual-size` always stores pre-processed object's size.

Reply:
Yes @aead ! `actual-size` should always be used to store pre-processed object's size.